src/test/java/seedu/address/logic/commands/EditStudentCommandTest.java

```diff
@@ -27,24 +27,25 @@
 import seedu.address.testutil.StudentBuilder;
 
 /**
- * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand) and unit tests for EditCommand.
+ * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand) and unit tests for
+ * EditStudentCommand.
```

Comment:
Perhaps it might be good to follow the same format as `DeleteStudentCommandTest` to be consistent?

```suggestion
 * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand) and unit tests for
 * {@code EditStudentCommand}.
```